### ISSUE TITLE
Error when running migrations.

### ISSUE THREAD

**Issue by SnidelyWhiplash** [balmasi/migrate-mongoose#14]
When I run `$ migrate` I get the following error:

```
module.js:338
    throw err;
          ^
Error: Cannot find module 'babel-runtime/core-js/object/assign'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/usr/local/lib/node_modules/migrate-mongoose/dist/cli.js:4:15)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
```

Any help?

**Comment by ertanden** [comment]
Had the same error. Update Node version. I used 6.9.4 LTS without a problem.

**Comment by balmasi** [comment]
I'll see if I can integrate babel-env to polyfill these for required environments

**Comment by SnidelyWhiplash** [comment]
@balmasi Any progress here? I created an [ugly fork](https://github.com/SnidelyWhiplash/migrate-mongoose) that just changes the dep, require call, and .babelrc...

I had issues with requiring the fork directly into my project though, due to the prepublish scripts and such. I ran that manually and finally got everything working as intended (though I did have to add `babel-runtime` as a dep too).

tl;dr I hacked it into submission, but on your end it should be a pretty simple drop-in.

Cheers.

**Comment by balmasi** [comment]
Hey all, I've removed support for babel in 4.0.0 so I'm afraid that makes pursuing this a dead end.

Thanks for raising it with me regardless!

I'd love to hear your opinions on what you think you still need babel for your migrations or not
